python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -348,6 +361,14 @@ def global_avgpooling(attrs, inputs, proto_obj):
                                                                 'pool_type': 'avg'})
     return 'Pooling', new_attrs, inputs
 
+def global_lppooling(attrs, inputs, proto_obj):
+    """Performs global lp pooling on the input."""
+    p_value = attrs['p'] if 'p' in attrs else 2
```

Comment:
p_value = attrs.get('p', 2)

Reply:
what is lp pooling?

Reply:
check here - https://mxnet.incubator.apache.org/api/python/ndarray/ndarray.html?highlight=poolin#mxnet.ndarray.Pooling